src/server/pom.xml

```diff
@@ -16,6 +16,7 @@
         <dropwizard.metrics.datadog.version>1.0.6</dropwizard.metrics.datadog.version>
         <cassandra.version>2.2.7</cassandra.version>
         <cucumber.version>1.2.5</cucumber.version>
+        <docker.directory>src/main/docker</docker.directory>
```

Comment:
This looks good if the `docker.directory` will be changed frequently. What was the use case you had in mind? Or was it just due to the reorg?

Reply:
This was kind of part of the reorg. It allows the following command to be run from the top-level of the repository.
```bash
mvn -f src/server/pom.xml clean package docker:build -Ddocker.directory=src/server/src/main/docker
```

Reply:
Ah, thanks for clarifying that for me.

Reply:
No worries.